plugins/restapi/v2/plugin.go

```diff
@@ -60,9 +60,13 @@ const (
 	// MIMEVendorIOTASerializer => bytes
 	RouteTransactionsIncludedMessage = "/transactions/:" + restapipkg.ParameterTransactionID + "/included-message"
 
-	// RouteMilestone is the route for getting a milestone by its milestoneIndex.
+	// RouteMilestoneByIndex is the route for getting a milestone by its milestoneIndex.
 	// GET returns the milestone.
-	RouteMilestone = "/milestones/:" + restapipkg.ParameterMilestoneIndex
+	RouteMilestoneByIndex = "/milestones/index/:" + restapipkg.ParameterMilestoneIndex
+
+	// RouteMilestone is the route for getting a milestone by its ID.
+	// GET returns the milestone.
+	RouteMilestone = "/milestones/milestoneId/:" + restapipkg.ParameterMilestoneID
```

Comment:
we said we wanted to remove `milestoneId` from the path

Reply:
oh, I just copied it from iota.go code, and this is based on our current TIP [swagger](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/stardust-api/tips/TIP-0025/core-rest-api.yaml)